lib/auth/tls_test.go

```diff
@@ -2054,16 +2055,16 @@ func TestGenerateCerts(t *testing.T) {
 		require.NoError(t, err)
 
 		_, diff := parseCert(userCerts.SSH)
-		require.Less(t, int64(diff), int64(iUser.TTL))
+		require.LessOrEqual(t, diff, maxSessionTTL)
```

Comment:
this test had a bug, the cert lifetime should not be limited to `iUser.TTL`, the correct limit is `maxSessionTTL` which is the `max_session_ttl` of the impersonated user. This is what the TLS cert expiry is compared against a few lines below